plugin/vscode/quick-lint-js/vscode/qljs-workspace.cpp

```diff
@@ -234,10 +243,29 @@ ::Napi::Value QLJS_Workspace::configuration_changed(
   ::Napi::Env env = info.Env();
 
   this->update_logging(env);
+  this->on_translator_changed(env);
 
   return env.Undefined();
 }
 
+void QLJS_Workspace::on_translator_changed(::Napi::Env env) {
+  Extension_Configuration config(env, this->vscode_);
+  bool is_snarky = config.get_snarky(env);
+
+  this->is_snarky_enabled_ = is_snarky;
+  if (is_snarky) {
+    this->translator_.use_messages_from_locale("en_US@snarky");
+  } else {
+    // TODO(#529): Use the locale from the VS Code configuration.
+    this->translator_.use_messages_from_source_code();
+  }
+  this->qljs_documents_.for_each(
+      [this, is_snarky, env](::Napi::Value value) -> void {
```

Comment:
Compiler was warning about `is_snarky` being an unused capture. I fixed it by using `[&]` instead.